detections/endpoint/windows_potential_appdomainmanager_hijack_artifacts_creation.yml

```diff
@@ -0,0 +1,78 @@
+name: Windows Potential AppDomainManager Hijack Artifacts Creation
+id: be19b369-fd0c-42be-ae97-c10b6c01638f
+version: 1
+date: '2025-12-10'
+author: Teoderick Contreras, Splunk
+status: production
+type: Anomaly
+description: The following analytic detects the creation of an .exe file along with its corresponding .exe.config and a .dll in the same directory, which is a common pattern indicative of potential AppDomain hijacking or CLR code injection attempts. This behavior may signal that a malicious actor is attempting to load a rogue assembly into a legitimate application's AppDomain, allowing code execution under the context of a trusted process.
+data_source:
+- Sysmon EventID 11
+search: |
+  | tstats `security_content_summariesonly` count min(_time) AS firstTime max(_time) AS lastTime from datamodel=Endpoint.Filesystem
+  where Filesystem.file_name IN ("*.exe", "*.exe.config", "*.dll") AND Filesystem.file_path IN
+    ("*\\windows\\fonts\\*", "*\\temp\\*", "*\\users\\public\\*", "*\\windows\\debug\\*","*\\Users\\Administrator\\Music\\*", "*\\Windows\\servicing\\*", "*\\Users\\Default\\*", "*Recycle.bin*", "*\\Windows\\Media\\*", "*\\Windows\\repair\\*", "*\\PerfLogs\\*", "*\\programdata\\*")
```

Comment:
ProgramData could be prone to FPs. In some of my test systems I already have `.exe` and corresponding `.exe.config` files created.